backend/app/api/src/helpers/SetupStepsUpdater.ts

```diff
@@ -0,0 +1,194 @@
+import {
+    Organization,
+    OrganizationRegistrationPeriod,
+    Platform,
+} from "@stamhoofd/models";
+import { SetupStepType, SetupSteps } from "@stamhoofd/structures";
+
+export class SetupStepUpdater {
+    private static readonly STEP_TYPE_OPERATIONS: Record<SetupStepType, (setupSteps: SetupSteps,
+        organization: Organization,
+        platform: Platform) => void> = {
+        [SetupStepType.Groups]: this.updateStepGroups,
+        [SetupStepType.Premises]: this.updateStepPremises
+    }
+
+    static async updateSetupStepsForAllOrganizationsInCurrentPeriod({
+        batchSize, stepTypes, platforms
+    }: { batchSize?: number, stepTypes?: SetupStepType[], platforms?: Platform[] } = {}) {
+
+        if(!platforms) {
+            platforms = await Platform.all();
+        }
+
+        for (const platform of platforms) {
+            const periodId = platform.periodId;
+
+            let lastId = "";
+
+            // eslint-disable-next-line no-constant-condition
+            while (true) {
+                const organizationRegistrationPeriods =
+                    await OrganizationRegistrationPeriod.where(
+                        {
+                            id: { sign: ">", value: lastId },
+                            periodId: periodId,
+                        },
+                        { limit: batchSize ?? 10, sort: ["id"] }
+                    );
+
+                if (organizationRegistrationPeriods.length === 0) {
+                    lastId = "";
+                    break;
+                }
+
+                for (const organizationRegistrationPeriod of organizationRegistrationPeriods) {
+                    const organizationId =
+                        organizationRegistrationPeriod.organizationId;
+                    if (!organizationId) {
```

Comment:
Voor zover ik weet is dit niet nullable, en dus overbodig, of zie ik iets over het hoofd?